client/src/modules/invoices/patientInvoice.html

```diff
@@ -110,25 +110,13 @@
             </button>
           </div>
 
-          <!-- "Add number of grid rows" input-group -->
-          <div class="input-group">
-            <span class="input-group-btn">
-              <button
-                id="btn-add-rows"
-                type="button"
-                class="btn btn-default"
-                ng-disabled="!PatientInvoiceCtrl.Invoice.recipient"
-                ng-click="PatientInvoiceCtrl.addItems(PatientInvoiceCtrl.itemIncrement)">
-                <i class="fa fa-plus-circle"></i> <span translate>FORM.BUTTONS.ADD</span>
-              </button>
-            </span>
-            <input
-              type="number"
-              class="form-control"
-              ng-model="PatientInvoiceCtrl.itemIncrement"
-              style="max-width : 40px;"
-              ng-disabled="!PatientInvoiceCtrl.Invoice.recipient">
-          </div>
+         <!-- "Component Add number of grid rows" input-group -->
+          <bh-add-item
+            item-increment = "PatientInvoiceCtrl.itemIncrement"
+            name = "add_item_invoice"
```

Comment:
Since this isn't ever going to be submitted as part of a form, there is no need for a `name` binding.  `name` is only useful for form validation or submission to servers.